webapp/app/actions.js

```diff
@@ -7,11 +7,14 @@ import Counter from '../../core/counter'
 import * as CognitoAuth from './cognitoAuth'
 
 import * as AppState from './appState'
-import { cancelDebouncedAction, debounceAction } from '../utils/reduxUtils'
 
 export const appPropsChange = 'app/props/change'
 export const appUserLogout = 'app/user/logout'
 
+export { throwSystemError } from '../loggedin/appErrors/actions'
+export { hideNotification, showNotificationMessage } from './appNotification/actions'
+export { showAppJobMonitor } from '../loggedin/appJob/actions'
```

Comment:
```showAppJobMonitor``` should not be exported from here, but imported directly from the components where it is dispatched